README.md

```diff
@@ -65,14 +67,16 @@ There are two files defining the dependencies:
     - `[tool.poetry.group.dev.dependencies]` lists the dependencies used for developing and testing.
 - `poetry.lock` enables deterministic installations, 
   by exactly pinning the versions of all (sub-)dependencies. 
-  This is done automatically, you must not edit `poetry.lock`.
+  This is done automatically and you must not edit `poetry.lock`.
 
 If you want to install a new package, 
-install it with `poetry add package`. This 
+install it with `poetry add package`.
 
-- installs the package (incl. sub-dependencies) in your virtual environment
-- adds the package to the section `[tool.poetry.dependencies]` of `pyproject.toml`
-- adds the pinned versions of the package and all sub-dependencies to `poetry.lock`
+This
+
+- Installs the package (incl. sub-dependencies) in your virtual environment,
+- Adds the package to the section `[tool.poetry.dependencies]` of `pyproject.toml`,
+- Adds the pinned versions of the package and all sub-dependencies to `poetry.lock`.
```

Comment:
I just wonder: is it correct to capitalize the start of bullet points, if they are part of a sentence? It seems counter-intuitive to me to write "This Installs ...", I would write "installs". 

I just wonder because I'd like to do it correct in the future :-)